extensions/grapher/src/com/google/inject/grapher/BindingEdge.java

```diff
@@ -22,6 +22,7 @@
  * Edge that connects an interface to the type or instance that is bound to implement it.
  *
  * @author phopkins@gmail.com (Pete Hopkins)
+ * @since 4.0
  */
 public class BindingEdge extends Edge {
```

Comment:
@sameb looks like this class was in 3.0, but was previously parameterized. i'll remove the since tag on it if that makes sense to you